sysconfigs/daint/compilers.yaml

```diff
@@ -89,21 +89,6 @@ compilers:
     - pgi/20.1.0
     environment: {}
     extra_rpaths: []
-- compiler:
-    spec: pgi@20.1.1
```

Comment:
I don't get why you want to remove pgi@20.1.1 I would personally remove pgi@20.1.0.

Reply:
PGI 20.1.1 does not exist.

Reply:
I don't care if it exists or not, on Daint when you load PrgEnv-pgi it is the compiler that is automatically loaded with the environement.

Reply:
Checked, indeed 20.1.1 is a default module. I'll change it then.